tests/jetty-test-session-common/src/main/java/org/eclipse/jetty/session/test/tools/RemoteInfinispanTestSupport.java

```diff
@@ -50,35 +49,39 @@ public class RemoteInfinispanTestSupport
     private static final Logger LOG = LoggerFactory.getLogger(RemoteInfinispanTestSupport.class);
     public RemoteCache<String, InfinispanSessionData> _cache;
     private final String _name;
-    public static RemoteCacheManager _manager;
     private static final Logger INFINISPAN_LOG =
             LoggerFactory.getLogger("org.eclipse.jetty.server.session.remote.infinispanLogs");
 
-    private static final String INFINISPAN_VERSION = System.getProperty("infinispan.docker.image.version", "14.0.25.Final");
+    private static final String INFINISPAN_VERSION = System.getProperty("infinispan.docker.image.version", "15.0.9.Final");
     private static final String IMAGE_NAME = System.getProperty("infinispan.docker.image.name", "infinispan/server") +
             ":" + INFINISPAN_VERSION;
 
-    private static final GenericContainer<?> infinispan = new GenericContainer<>(IMAGE_NAME);
-
-    static
+    private static final class Holder
```

Comment:
What's the benefit of doing it this way vs having `static`s?

Reply:
easier to read :)